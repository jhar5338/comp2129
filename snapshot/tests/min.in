min a
set a 1 0 -1
get a
min a
set b
min b
bye
