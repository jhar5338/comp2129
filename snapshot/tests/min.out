> no such key

> ok

> [1 0 -1]

> -1

> ok

> nil

> bye
